### PR TITLE
working endpoints crm

### PR DESCRIPTION
**Pull request by stivni**

### CODE REVIEW

File: src/02-crm/companies.apib [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+            + business_type: `NV` (string)
+            + vat_number: `BE0899623035` (string)
+            + local_business_number: `63326426` (string)
+            + email: `info@piedpiper.eu` (string)
+            + invoice_email: `invoicing@example.com` (string, optional)
+            + telephones (array[Telephone])
+            + website: `https://piedpiper.com` (string)
+            + avatar (string)
+            + address (Address)
+            + delivery_address (Addressee)
```
Comment (jenssegers) [inline src/02-crm/companies.apib]: Missing

File: src/02-crm/contacts.apib [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+Add a new contact.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + first_name: `John` (string, required)
+        + last_name: `Smith` (string, required)
+        + email: `john@example.com` (string, required)
+        + salutation: `Mr` (string, optional)
+        + email: `john@example.com` (string, required)
```
Comment (jenssegers) [inline src/02-crm/contacts.apib]: double `email`

File: src/02-crm/contacts.apib [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+        + telephones (array[Telephone])
+        + website: `http://example.com` (string, optional)
+        + address (Address, optional)
+        + delivery_address (Addressee, optional)
```
Comment (jenssegers) [inline src/02-crm/contacts.apib]: Should we include `avatar`?

Comment (mrnnn) [reply src/02-crm/contacts.apib]: It's easy and pretty much free, right? Let's do it, at least in the info?

File: src/02-crm/companies.apib (hunk 1 above)
Comment (mrnnn) [reply src/02-crm/companies.apib]: They are there now, but are we changing this to an array before releasing? or not?

**Review by marijnvandevoorde — APPROVED** [review]
LGTM and iterate? Only blocking thing is the addresses

File: src/02-crm/companies.apib (hunk 1 above)
Comment (jenssegers) [reply src/02-crm/companies.apib]: I would prefer to release with an array.

**Comment by stivni** [comment]
@jenssegers made another PR

**Pull request merged**